op-chain-ops/genesis/testdata/test-deploy-config-full.json

```diff
@@ -40,12 +40,15 @@
   "baseFeeVaultRecipient": "0x42000000000000000000000000000000000000f5",
   "l1FeeVaultRecipient": "0x42000000000000000000000000000000000000f6",
   "sequencerFeeVaultRecipient": "0x42000000000000000000000000000000000000f7",
+  "operatorFeeVaultRecipient": "0x42000000000000000000000000000000000000f8",
   "baseFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
   "l1FeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
   "sequencerFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
+  "operatorFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
```

Comment:
Should be able to delete per https://github.com/ethereum-optimism/optimism/pull/12166/files#r1941779240